File: branches/Medusa/Source/Fantasy.Jobs/Management/JobManagerAccessor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.Jobs.Management
{
    public class JobManagerAccessor : MarshalByRefObject
    {

        private static IJobManager _manager;

        public IJobManager GetJobManager()
        {
            return _manager ?? JobManager.Default; 
        }

        public void SetJobManager(JobManager manager)
        {
            _manager = manager;
        }
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Jobs/Management/JobManagerAccessor.java
package Fantasy.Jobs.Management;

public class JobManagerAccessor extends MarshalByRefObject
{

	private static IJobManager _manager;

	public final IJobManager GetJobManager()
	{
		return (_manager != null) ? _manager : JobManager.getDefault();
	}

	public final void SetJobManager(JobManager manager)
	{
		_manager = manager;
	}
}